go.mod

```diff
@@ -1,23 +1,19 @@
 module github.com/SafetyCulture/s12-proto
 
-go 1.18
+go 1.22
 
 require (
 	github.com/gofrs/uuid v3.3.0+incompatible
-	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.5.2
+	github.com/gogo/protobuf v1.3.2
 	github.com/icrowley/fake v0.0.0-20180203215853-4178557ae428
-	google.golang.org/protobuf v1.28.1
-)
-
-require (
-	github.com/stretchr/testify v1.7.0 // indirect
-	golang.org/x/crypto v0.1.0 // indirect
-	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
-	sc-go.io/pkg/jwtclaims v1.7.0 // indirect
+	google.golang.org/protobuf v1.34.2
 )
 
 require (
 	github.com/corpix/uarand v0.1.1 // indirect
-	golang.org/x/text v0.4.0
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/google/go-cmp v0.5.6 // indirect
+	github.com/stretchr/testify v1.7.0 // indirect
+	golang.org/x/text v0.18.0
```

Comment:
Can we move this to the direct block above?

Reply:
sure